runtime/signals.c

```diff
@@ -139,6 +142,7 @@ CAMLexport void (*caml_enter_blocking_section_hook)(void) =
 CAMLexport void (*caml_leave_blocking_section_hook)(void) =
    caml_leave_blocking_section_default;
 
+CAMLno_tsan /* The read of [caml_something_to_do] is not synchronized. */
```

Comment:
I'm not sure about this line?
At least the comment is referring to trunk rather than Multicore.

Reply:
Yes, this is left over from trunk and doesn't make sense here. I'll remove.